### PR TITLE
Add a bio for @christine-erickson

### PR DESCRIPTION
**Pull request by christine-erickson**
This pull request adds a bio for @christine-erickson as requested in #9. 

@chris-erickson : This is my first pull request, I figured you would want to be involved.  I would :heart:  your opinion.

### CODE REVIEW

File: docs/christine-erickson.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Christine-Erickson
+
+**Name: Christine Erickson**
+
+**Occupation: Firmware Engineer**
+
+**Location: Milwaukee**
```
Comment (christine-erickson) [inline docs/christine-erickson.md]: Why don't you add WI, USA, Earth?

File: docs/christine-erickson.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Christine-Erickson
```
Comment (christine-erickson) [inline docs/christine-erickson.md]: What about your middle name?

File: docs/christine-erickson.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+**Name: Christine Erickson**
```
Comment (christine-erickson) [inline docs/christine-erickson.md:3]: What about your middle name?

File: docs/christine-erickson.md (hunk 1 above)
Comment (LostThought) [inline docs/christine-erickson.md]: What state is that in?

**Comment by LostThought** [comment]
Update your file :imp: :fire:

File: docs/christine-erickson.md (hunk 1 above)
Comment (christine-erickson) [inline docs/christine-erickson.md]: @LostThought, it's in Wisconsin :beer: :cow:  ..... but I'll clarify.

**Comment by christine-erickson** [comment]
:+1: Looks good to me.

:shipit:

**Comment by chris-erickson** [comment]
This info looks valid.  Did you write unit tests?

:rocket:

**Comment by christine-erickson** [comment]
Hi @chris-erickson.  Thanks for your comment, it is very appreciated.  But to answer your question, no.

:hand:

**Pull request merged**